cmd/create/execution_util.go

```diff
@@ -89,7 +92,10 @@ func createExecutionRequestForTask(ctx context.Context, taskName string, project
 		Version:      task.Id.Version,
 	}
 
-	return createExecutionRequest(id, inputs, securityContext, authRole), nil
+	labels := executionConfig.Labels
+	annotations := executionConfig.Annotations
```

Comment:
We will need a way to read these parameters from file too. check readConfigAndValidate